docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+This _audience member_ can be a developer who is generally speaking is new to JavaScript programming or a developer who is new to programming with Strapi. **Tutorials** are for the newer programmer. More experienced developers, who are new to Strapi, find the **How-tos** better suited to their experience, but they can also follow tutorials.
+
+Examples include the [Building a Static Blog using Gatsby and Strapi](https://blog.strapi.io/building-a-static-website-using-gatsby-and-strapi/) tutorial and the [Cooking a Deliveroo Clone](https://blog.strapi.io/strapi-next-setup/) tutorial.
+
+### How-to's
+
+A **How-to** is a **step-by-step guide showing how** to solve a specific problem or accomplish a specific task. These guides provide concise action steps and assume a general understanding of the context by the audience.
+
+This _audience member_ is a developer who is newer to programming with Strapi. General knowledge of the technology covered is assumed. Here the **How-tos** are meant to show how to do things with Strapi, without necessarily explaining the theory.
+
+Examples include the [Digital Ocean Deployment Guide](/3.0.0-beta.x/guides/deployment.html#digital-ocean) and [How to install MongoDB Locally](/3.0.0-beta.x/guides/databases.html#install-mongodb-locally).
+
+### Reference topics
+
+**Reference topics** are subjects that **are resources** to bookmark and used as a reference. Developers can be expected to refer to Reference topics regularly throughout the development of Strapi based projects.
+
+This _audience member_ is a developer who is already developing projects with Strapi. **Reference topics** are meant to be used repetitively as an up-to-date and managed resource for recipes, best practices, and sample code which can be copy/pasted.
+
+Examples include the [Models Guide](/3.0.0-beta.x/guides/models.html#how-to-create-a-model) and the [Controllers Guide](/3.0.0-beta.x/guides/controllers.html).
+
+### Explanations
+
+**Explanations** cover why something was done and provide an understanding of the choices made through the course of the development of Strapi. These are generally written as **Articles** and found in the [Strapi Blog](https://blog.strapi.io).
+
+This _audience member_ is a developer who is already developing projects with Strapi or who is studying whether Strapi is a good fit for their project(s). **Explanations** are meant to explain different choices in the Roadmap, overall direction, and how certain features generally work.
+
+Examples include [Why we split the management of the Admin user and the Front-end Users](https://blog.strapi.io/why-we-split-the-management-of-the-admin-users-and-end-users/) and [Restricting Permissions in Strapi Admin](https://blog.strapi.io/admin-permissions/).
```

Comment:
We should remove the Restricting Permissions article since it was made for the alpha and doesn't work anymore for the beta.

Reply:
DONE.